build/pause/Dockerfile.Rhel

```diff
@@ -1,11 +1,11 @@
-FROM registry.ci.openshift.org/ocp/builder:ubi8.art AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9 AS builder
```

Comment:
@umohnani8 since you were changing this last time, ptal

Reply:
LGTM, thanks for the ping!